support-frontend/assets/pages/[countryGroupId]/components/componentContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { from, space } from '@guardian/source/foundations';
+import type { ReactNode } from 'react';
+import { Container } from 'components/layout/container';
+
+const headingContentContainer = css`
+	${from.tablet} {
+		min-height: 450px;
+	}
+`;
+
+const container = css`
+	> div {
+		min-height: 480px;
+		${from.tablet} {
+			min-height: 432px;
+		}
+		padding: ${space[5]}px 10px;
+		${from.mobileLandscape} {
+			padding-left: ${space[5]}px;
+			padding-right: ${space[5]}px;
+		}
+		${from.desktop} {
+			max-width: 940px;
+		}
+	}
+`;
+
+export type ComponentContainerProps = {
+	children: ReactNode;
+	borderColor?: string;
+	sideBorders?: boolean;
+	topBorder?: boolean;
+	cssOverrides?: SerializedStyles;
+};
+
+export function ComponentContainer({
```

Comment:
This new `ComponentContainer` has a bit of a generic name, I don't it's clear the context / purpose of this component? 

I can see from your other PRs https://github.com/guardian/support-frontend/pull/6578 that it's designed to be reused elsehwhere on the Guardian Light landing page? Is this the only use cases for it? If so I wonder if 

1) Whether it needs to be a standalone module/file vs just defining the componeny inside the Guardian Light landing page ? If we decide it a standalone module/file is required then...
2) Should we rename it to indicate that it's use for Guardian Light landing page components only?

Reply:
I also wonder whether a nested directory inside `/components` for "Guardian Light" specific components might be a neater way of organising the files inside the `pages/[countryGroupId]/components/` directory? EG.

`pages/[countryGroupId]/components/`
- Components shared across the `checkout.tsx`, `oneTimeCheckout.tsx`, `guardianLightLanding.tsx` pages

`pages/[countryGroupId]/components/checkout`
- Components shared specific to the `checkout.tsx` page

`pages/[countryGroupId]/components/oneTimeCheckout`
- Components shared specific to the `oneTimeCheckout.tsx` page

`pages/[countryGroupId]/components/guardianLightLanding`
- Components shared specific to the `guardianLightLanding.tsx` page

Having everything in `/components` makes it hard to know how modules are intended to be used.

Reply:
There may be other ways to restructure the directory as well.

- pages/[countryGroupId]
  - /components
  -  /checkout
     - checkout.tsx
     - /components
   - /oneTimeCheckout
     - oneTimeCheckout.tsx
     - /components
   - /guardianLightLanding
     - guardianLightLanding.tsx
     - /components

Reply:
Done the folder reorganise (thanks for signing off).  Have simplified / removed usage of the shared generic `componentContainer.tsx` now too. Think that's those 2 review items covered.